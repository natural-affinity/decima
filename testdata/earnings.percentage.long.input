decima --percentage 12.5 80
