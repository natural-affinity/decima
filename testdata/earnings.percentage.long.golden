Usage:
    decima [--percent p] [--extra e] [--breakdown] <earnings>...
    decima --help
    decima --version
